pkg/controller/backup_sync_controller.go

```diff
@@ -107,4 +111,30 @@ func (c *backupSyncController) run() {
 			}
 		}
 	}
+
+	err = deleteUnused(c, cloudBackups)
+	if err != nil {
+		c.logger.WithError(errors.WithStack(err)).Error("Error deleting unused backup from Kubernetes")
+	}
+}
+
+// deleteUnused deletes a backup from Kubernetes if it does not exist in object storage
+func deleteUnused(c *backupSyncController, cloudBackups map[string]*api.Backup) error {
+	// Backups from Kubernetes
+	bl, err := c.client.Backups(c.namespace).List(metav1.ListOptions{})
```

Comment:
We'll want to switch to using a backup lister here, rather than a client.  Listers/informers are shared caches that can be used by many different controllers, that reflect the contents of the Kubernetes API, and are used to minimize load on the API server.  Take a look at the restore_controller, for a good example of using a `backupLister`.

Reply:
@skriss should I include the `backupInformer.Lister()` as a a param for the initialization of the sync_controller? Else I'm trying to see how to access that element since it resides in a different controller.